app/src/main/java/com/kms/cura/view/LoginActivity.java

```diff
@@ -0,0 +1,94 @@
+package com.kms.cura.view;
+
+import android.content.pm.ActivityInfo;
+import android.content.res.Configuration;
+import android.graphics.Paint;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.kms.cura.R;
+
+public class LoginActivity extends AppCompatActivity implements TextWatcher {
+
+    EditText email;
+    EditText password;
+    Button forgotPasswordButton,loginButton,createAccountButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        initView();
+    }
+
+    /**
+     * Init view with entity object
+     */
+    public void initView(){
+        //Set forgot paswword button style
+        forgotPasswordButton = (Button) findViewById(R.id.button_LoginUI_ForgotPassword);
+        forgotPasswordButton.setPaintFlags(forgotPasswordButton.getPaintFlags() | Paint.UNDERLINE_TEXT_FLAG);
+        forgotPasswordButton.setTransformationMethod(null);
+
+        //Set create account button style
+        createAccountButton = (Button) findViewById(R.id.button_LoginUI_CreateAccount);
+        createAccountButton.setTransformationMethod(null);
+
+        //Set login button style
+        loginButton = (Button) findViewById(R.id.button_LoginUI_Login);
+        loginButton.setTransformationMethod(null);
+        //Fix orientation
+        this.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        //Add TextChangedListener
+        email = (EditText) findViewById(R.id.editText_LoginUI_Email);
+        password = (EditText) findViewById(R.id.editText_LoginUI_Password);
+
+        email.addTextChangedListener(this);
+        password.addTextChangedListener(this);
+    }
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        // ignore orientation/keyboard change
+        super.onConfigurationChanged(newConfig);
+    }
+
+    @Override
+    public void beforeTextChanged(CharSequence s, int start,int count, int after){
+
+    }
+    /*  Primative check for email and password as soon as the user type in something
+    *   Disable the login button if either condition is invalid nad display an error message accordingly*/
+    @Override
+    public void onTextChanged(CharSequence s, int start, int before, int count){
+        if(!isEmailValid(email.getText().toString())){
+            email.setError("Invalid Email");
```

Comment:
Should create a string resource file (for example strings_error.xml) for error string like this.